Makefile

```diff
@@ -0,0 +1,30 @@
+BOARD ?= m3
+
+.PHONY: build-docker-image build-docker-image-tests setup-ude-rules \
+	    test integration-test
+
+build-docker-image:
+	docker build -t iot-lab-gateway .
+
+build-docker-image-test: build-docker-image
+	docker build -t iot-lab-gateway-tests tests
+
+setup-udev-rules:
+	sudo cp bin/rules.d/* /etc/udev/rules.d/.
+	sudo udevadm control --reload-rules
+
+test:
+	docker run -t --rm \
+		-v $(PWD):/shared \
+		-e LOCAL_USER_ID=`id -u $(USER)` \
+		iot-lab-gateway-tests tox
+
+integration-test:
+	echo $(BOARD) > $(PWD)/tests_utils/cfg_dir/board_type
```

Comment:
This is problematic because you're modifying the code source when testing, I would create the cfg_dir in /tmp maybe, copied from tests_utils/cfg_dir and modify it there

Reply:
I know, but I find this very when testing.

> I would create the cfg_dir in /tmp maybe, copied from tests_utils/cfg_dir and modify it there

Good idea, I'll do that.